<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertListPopUp</class>
 <widget class="QDialog" name="AlertListPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Alerts</string>
  </property>
  <widget class="QPlainTextEdit" name="FilePath">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="plainText">
    <string>C:\Qt\test.txt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ICalander Alert Save Location: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Generated Alerts:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <item>
       <property name="text">
        <string>1st Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="UpdateButton">
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string># of character right of date.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="PostRX">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="plainText">
       <string>30
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string># of character left of date.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="PreRX">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>21</height>
       </size>
      </property>
      <property name="plainText">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
